systems/registry/network/cmd/server/main.go

```diff
@@ -51,7 +51,7 @@ func initConfig() {
 func initDb() sql.Db {
 	log.Infof("Initializing Database")
 	d := sql.NewDb(serviceConfig.DB, serviceConfig.DebugMode)
-	err := d.Init(&db.Org{}, &db.Network{}, &db.Network{})
+	err := d.Init(&db.Org{}, &db.Network{}, &db.Site{})
```

Comment:
Again, this is weird. That line was already fixed in upstream registry-system base branch.

Reply:
No it get missed somehow, this branch and subscriber-system branch is up-to-date with `main`.

Reply:
Brackley requested me add this line in my PR

Reply:
This is so weird!